universe/Pathfinder.h

```diff
@@ -101,8 +101,20 @@ class FO_COMMON_API Pathfinder {
       * \a empire_id == ALL_EMPIRES.
       * \throw std::out_of_range This function will throw if the  system
       * ID is out of range. */
+    //TODO empire_id is never set to anything other than self, which in
+    //the AI's is the same as ALL_EMPIRES
     std::multimap<double, int>              ImmediateNeighbors(int system_id, int empire_id = ALL_EMPIRES) const;
 
+    /** Returns the \p near and \p far objects from the \p candidates that are
+        within \p jumps of the \p others.
+        The \p candidates are modified and transfered to the \near vector.*/
```

Comment:
confused... do objects also get moved into `far`? is anything left in `candidates`?

don't like `others` as a parameter name... too vague. perhaps `from_objects` ?